cherrypy/lib/static.py

```diff
@@ -28,6 +29,23 @@ def _setup_mimetypes():
 _setup_mimetypes()
 
 
+def _make_content_disposition(disposition, file_name):
+    """Create HTTP header for downloading a file with a UTF-8 filename.
+
+    This function implements the recommendations of :rfc:`6266#appendix-D`.
+    See this and related answers: https://stackoverflow.com/a/8996249/2173868.
+    """
+    ascii_name = (
+        unicodedata.normalize('NFKD', file_name).
+        encode('ascii', errors='ignore').decode()
+    )
+    header = u'{}; filename="{}"'.format(disposition, ascii_name)
+    if ascii_name != file_name:
+        quoted_name = urllib.parse.quote(file_name)
+        header += u'; filename*=UTF-8\'\'{}'.format(quoted_name)
```

Comment:
Since we don't support Python2, no need to use u-prefix in string literals
```suggestion
    header = '{}; filename="{}"'.format(disposition, ascii_name)
    if ascii_name != file_name:
        quoted_name = urllib.parse.quote(file_name)
        header += '; filename*=UTF-8\'\'{}'.format(quoted_name)
```